src/commons/scheduler.py

```diff
@@ -96,3 +100,36 @@ def schedule_test_plan(test_plan: str, test_plan_values: dict, common_params: di
     finally:
         loop.stop()
         LOGGER.critical("%s terminated", process_name)
+
+
+def schedule_test_status_update(parsed_input, corio_start_time):
+    """
+    Schedule the test status update.
+
+    :param parsed_input: Dict for all the input yaml files.
+    :param corio_start_time: Start time for main process.
+    """
+    execution_time_list = [corio_start_time]
+    for _, test_d in parsed_input.items():
+        for _, test_data in test_d.items():
+            exec_time = corio_start_time + test_data['start_time'] + test_data['min_runtime']
+            if exec_time not in execution_time_list:
+                execution_time_list.append(exec_time)
+    for run_time in execution_time_list:
+        sched.add_job(lambda: log_status(parsed_input=parsed_input,
```

Comment:
@DhobaleVishal I don't think this will reflect the correct status of run. This will be only executed at start of each test and not periodically.

Reply:
Agree, updated it as per our discussion. status will be updated after 30mins.